packages/office-ui-fabric-react/src/components/Stack/Stack.types.ts

```diff
@@ -135,6 +135,11 @@ export interface IStackProps
  * {@docCategory Stack}
  */
 export interface IStackTokens {
+  /**
+   * Defines the spacing between Stack children.
+   * The property is specified as a value for 'row gap', followed optionally by a value for 'column gap'.
+   * If 'column gap' is omitted, it's set to the same value as 'row gap'.
```

Comment:
Would using `@defaultValue` apply here given your doc comment?

Reply:
Hmmmm, I wonder what the `@defaultValue` would be though. Maybe just `0`? We could certainly add it if needed.